build/ci/.azure-devops-uap.yml

```diff
@@ -20,6 +20,7 @@ jobs:
 
   steps:
   - checkout: self
+    clean: true
```

Comment:
@jeromelaban I see this job (and also many other jobs) are using `gitversion.yml`, probably unnecessarily. Do you know for which jobs we can remove the use of `gitversion.yml`? Is it any non-packaging job? The removal of `gitversion.yml` will allow us to do shallow fetch which will be faster.

Reply:
Non packaging jobs don't need gitversion (it's not supposed to be there, iirc). For the others, full clone and tags are required.